### ISSUE TITLE
New Rule for AWS::ECS::CapacityProvider resource

### ISSUE THREAD

**Issue by raknass** [aws-cloudformation/cfn-python-lint#1745]
*cfn-lint version: 0.24.3

*Description of issue.*
Fails to validate AWS:ECS:CapacityProvider resource. Cloudformation support added recently.
```
E3002 Invalid Property Resources/Cluster/Properties/CapacityProviders
E3001 Invalid or unsupported Type AWS::ECS::CapacityProvider for resource CapacityProvider
```

```
CapacityProvider:
    Type: AWS::ECS::CapacityProvider
    Properties:
      AutoScalingGroupProvider:
        AutoScalingGroupArn: !Ref AutoScalingGroup
        ManagedScaling:
          Status: ENABLED
          TargetCapacity: 100
          MinimumScalingStepSize: 1
          MaximumScalingStepSize: 100
        ManagedTerminationProtection: DISABLED
```

**Comment by PatMyron** [comment]
[`cfn-lint 0.24.3`](https://github.com/aws-cloudformation/cfn-python-lint/releases/tag/v0.24.3) was released in Sep 2019

```bash
pip3 install cfn-lint --upgrade
```
